identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -1,22 +1,22 @@
 @import form.IdFormHelpers._
-@import model.{EmailSubscription, EmailSubscriptions}
+@import model.{EmailNewsletter, EmailNewsletters}
 @import views.html.fragments.form.radioField
 @import views.support.RenderClasses
 @import views.support.`package`.Seq2zipWithRowInfo
 
 @import common.LinkTo
 
-@(page: model.Page, emailPrefsForm: Form[EmailPrefsData], formActionUrl: String, emailSubscriptions: EmailSubscriptions)(implicit request: RequestHeader, messages: play.api.i18n.Messages, context: model.ApplicationContext)
+@(page: model.Page, emailPrefsForm: Form[EmailPrefsData], formActionUrl: String, emailSubscriptions: List[String], availableLists: EmailNewsletters)(implicit request: RequestHeader, messages: play.api.i18n.Messages, context: model.ApplicationContext)
 
-@emailListCategoryList(theme: String, subscriptions: List[EmailSubscription], isActive: Boolean) = {
+@emailListCategoryList(theme: String, subscriptions: List[EmailNewsletter], isActive: Boolean) = {
```

Comment:
Since you've very helpfully distinguished the notion of the `EmailNewsletter` itself from subscriptions (which define which newsletters a user is subscribed to), perhaps we could change this parameter name too? From `subscriptions` to `newsletters`. And similarly `subscription` to `newsletter` in the `.map` below